pkg/mount/custom_mount.go

```diff
@@ -46,7 +50,7 @@ func NewCustomMounter(
 }
 
 // Load mount table
-func (c *CustomMounterHandler) Load(devPrefixes []string) error {
+func (c *CustomMounterHandler) Load(devPrefixes []*regexp.Regexp) error {
```

Comment:
nit: if using regex, do we still call this devPrefixes?

Reply:
Good point, I don't think so.